content/launch-keyboard.md

```diff
@@ -65,6 +65,7 @@ In the <u>Keyboard Configurator</u>, you can change keyboard back light color an
 
 We have documentation that goes over changing the keycaps, switches, and more on our Tech Docs site:
 
-- [Launch](https://tech-docs.system76.com/models/launch_1/repairs.html)
+- [Launch launch_1](https://tech-docs.system76.com/models/launch_1/repairs.html)
+- [Launch launch_2](https://tech-docs.system76.com/models/launch_2/repairs.html)
```

Comment:
"Launch launch_1" seems weird. Maybe should be "Launch (launch_1)" or just "launch_1"? If the latter, we probably want to change "Launch Lite" to "launch_lite_1" and "Launch Heavy" to "launch_heavy_1"

Reply:
I put them in parenthesis and added them for the other models with 6db34ba. This matches the formatting used in the Tech Docs sidebar (other than the order.)

![image](https://user-images.githubusercontent.com/7199422/232138402-bf5a6c01-9c1d-486c-8fc9-2964802038ce.png)

Reply:
Looks good!